### ISSUE TITLE
Fresh install error message: Cannot assign null to property Junges\ACL\AclRegistrar::$permissionClass of type string

### ISSUE THREAD

**Issue by QuindoHL** [mateusjunges/laravel-acl#256]
When running the following command during setup I get this error.  I am not sure if it is related to the other issue.

```
vagrant@homestead:~/Code/projectdirectory$ php artisan vendor:publish --provider="Junges\ACL\Providers\ACLServiceProvider"
Can't locate path: <0>
Copied Directory [/vendor/mateusjunges/laravel-acl/resources/views] To [/resources/views/vendor/junges/acl]
Copied Directory [/vendor/mateusjunges/laravel-acl/resources/lang] To [/resources/lang/vendor/acl]

```

When attempting to browser the newly created laravel project I get the following error.

```
Junges \ ACL \ AclRegistrar : 34 __construct

Cannot assign null to property Junges\ACL\AclRegistrar::$permissionClass of type string

```

I am assuming that the issue is that the config file is not getting published but I have not deep dived into it yet.  I wanted to alert you about this real quick before I got pulled off onto a different.

**Comment by mateusjunges** [comment]
Hi! Thanks for reporting, I'll look into it later today

**Comment by mateusjunges** [comment]
@QuindoHL fixed and released in v4.1.1. Again, thanks for reporting this.
